src/NuGet.Clients/NuGet.PackageManagement.UI/Resources/PackageIconMonikers.tt

```diff
@@ -1,10 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
+<#@ template hostspecific="true" #>
+<#@ parameter type="System.String" name="PMUIAssemblyVersion" #>
 <ImageManifest
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xmlns:xsd="http://www.w3.org/2001/XMLSchema"
   xmlns="http://schemas.microsoft.com/VisualStudio/ImageManifestSchema/2014">
   <Symbols>
-    <String Name="Resources" Value="/NuGet.PackageManagement.UI;Component/" />
+    <String Name="Resources" Value="/NuGet.PackageManagement.UI;v<#= PMUIAssemblyVersion #>;31bf3856ad364e35;Component/" />
```

Comment:
Did you try using assembly version 0.0.0.0 to see if our binding redirects cause it to load the correct assembly version without having to regenerate this file every build?

Reply:
Great idea, but no, it doesn't seem to work.
```
The operation failed.
Bind result: hr = 0x80070002. The system cannot find the file specified.

Assembly manager loaded from:  C:\Windows\Microsoft.NET\Framework64\v4.0.30319\clr.dll
Running under executable  C:\Program Files\Microsoft Visual Studio\2022\Preview\Common7\IDE\devenv.exe
--- A detailed error log follows. 

=== Pre-bind state information ===
LOG: DisplayName = NuGet.PackageManagement.UI, Version=0.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35
 (Fully-specified)
LOG: Appbase = file:///C:/Program Files/Microsoft Visual Studio/2022/Preview/Common7/IDE/
LOG: Initial PrivatePath = NULL
LOG: Dynamic Base = NULL
LOG: Cache Base = NULL
LOG: AppName = devenv.exe
Calling assembly : (Unknown).
===
```